### DISCUSSION TITLE
Deepgram 2022 Reading List

### DISCUSSION THREAD

**Discussion by jkrugman** [deepgram/community#84] [General help]
We love growing at Deepgram, and part of that is thrown reading things that help us to better understand the people around us. Here’s a list of some of the books our team is reading:

[Build: An Unorthodox Guide to Making Things Worth Making](https://amzn.to/3h4QRbc) - By Tony Fadell

[Dare to Lead: Brave Work. Tough Conversations. Whole Hearts](https://amzn.to/3PanKQ3). - By Brene Brown 

[How Google Works](https://amzn.to/3VG54u8) - Eric Schmidt 

[Good to Great ](https://www.amazon.com/dp/0066620996/ref=cm_sw_r_as_gl_api_gl_i_BECNYWHM56KQ65F5DV4Q?linkCode=ml2&tag=jordankrugman-20)- Jim Collins 

[Crossing the Chasm](https://www.amazon.com/Crossing-Chasm-3rd-Disruptive-Mainstream/dp/0062292986/ref=sr_1_1?keywords=crossing+the+chasm&qid=1670443546&sprefix=crossing+the+%2Caps%2C165&sr=8-1) -  Geoffrey A. Moore 

[Zero to One](https://amzn.to/3iJAp0v) - Peter Thiel 

[The Hard Thing about Hard Things](https://amzn.to/3YhVHCL) - Ben Horowitz

[Start with Why](https://amzn.to/3HksqkC) - Simon Sinek 

[Making of a Manager](https://amzn.to/3VGkZIG) - Julie Zhou

[Growth](https://amzn.to/3PaykGX) - Vaclav Smil

[High Output Management ](https://amzn.to/3Fd7Pfd)- Andrew Grove

[Art of Doing Science and Engineering](https://amzn.to/3FxhQFI) - Richard Hamming

[Radical Candor](https://amzn.to/3VEBUeZ) - Kim Scott 

[Thanks for the Feedback](https://amzn.to/3YgmAH8) - Douglas Stone 

[The Happiness Advantage](https://amzn.to/3BgHTyl) - Shawn Achor

**Comment by BekahHW** [comment]
Radical Candor is one of my all-time favorites. I'll add the classic, Lean Startup, and [The Business of Belonging](https://www.amazon.com/Business-Belonging-Community-Competitive-Advantage/dp/1119766125) by David Spinks as well.

**Comment by josencf** [comment]
Fantastic list!

The Hard Thing about Hard Things and How Google Works were suuuuper insightful! 

And if I may append to the list, here's a few books I read last year that I loved (no spoilers):

**Nonfiction**
- [Never Split the Difference](https://www.amazon.com/Never-Split-Difference-Negotiating-Depended/dp/0062407805) by Chris Voss and Tahl Raz - This book taught me incredible negotiation skills that anyone can pick up. I was able to talk my way into a few discounts on meals, extra deals on clothes, and even some first class flights thanks to the advice in this book!
- [The Like Switch](https://www.amazon.com/The-Like-Switch-audiobook/dp/B00RKI81A6/ref=sr_1_1?crid=34LPULGZXYQGU&keywords=the+like+switch&qid=1674251950&s=books&sprefix=the+like+swit%2Cstripbooks%2C153&sr=1-1) by Jack Schafer - This book, much like *Never Split the Difference* was written by an ex-FBI agent. It focuses on body language and voice, giving tips on how to get anyone on your side. The book thankfully lacks an ego, and its advice really helped me in my presentations!
- [What the Dog Saw](https://www.amazon.com/What-Dog-Saw-Other-Adventures/dp/0316076201/ref=tmm_pap_swatch_0?_encoding=UTF8&qid=1674252257&sr=1-1) by Malcolm Gladwell - Yes, we can debate for hours on whether Gladwell's conclusions (ex: The 10,000 hours rule) are true. But I read Gladwell more to learn from his writing style. And this book is the perfect vehicle for that. If you ever want to write blog posts or articles that "play" with your audience—that is, feeds them information in a very calculated pace such that the conclusion is as mind-blowing as possible—Gladwell is the writer you're looking for. *What the Dog Saw* contains a list of all of Gladwell's New Yorker articles from the 1980s to present day.

**Fiction**
- [Beautiful World, Where Are You](https://www.amazon.com/Beautiful-World-Where-Are-You/dp/1250859042/ref=tmm_pap_swatch_0?_encoding=UTF8&qid=1674252322&sr=1-1) by Sally Rooney - A story about young adults trying to find themselves and navigate the complexities of a tough world, from work to romance to scandals. Rooney's voice is unmatched in its beauty and uniqueness. Whenever I go through tough times at my age, I think to Rooney's books and remember that I'm not alone.
- [From Blood and Ash](https://www.amazon.com/Blood-Ash-Jennifer-L-Armentrout/dp/1952457769/ref=tmm_pap_swatch_0?_encoding=UTF8&qid=1674252477&sr=1-1) by Jennifer L. Armentrout - This one is for the fantasy romance lovers out there. If you know what ACoTaR stands for, if the name "Briar University" sounds familiar to you, or if you are generally an enjoyer of the fantasy-romance genre, then you'll love this book, too. 
- [My Year of Rest and Relaxation](https://www.amazon.com/My-Year-Rest-Relaxation-Novel/dp/0525522131/ref=sr_1_1?crid=3BGYEI2EVQ8S5&keywords=my+year+of+rest+and+relaxation&qid=1674252616&s=books&sprefix=my+year+of+rest+and+relaxation%2Cstripbooks%2C134&sr=1-1) by Ottessa Moshfegh - This book got pretty trendy recently, especially on TikTok. Personally, I'm biased because I absolutely adore the author. She and I had the same writing mentor back in college. Nevertheless, the story itself is wonderful. A girl who has it all—a well-paying New York City job, pretty privilege, and an Ivy League degree—is unhappy with her life. It turns out that the things we'd envy her for only represent the surface of her life. This book chronicles her journey (or, rather, attempt) to putting herself asleep for a whole year via Benadryl, Ambien, Lunesta, and any other sleepy drug you can think of. It's a wonderful look at depression and mental health.
